src/page-modules/contact/travel-guarantee/form-selector.tsx

```diff
@@ -0,0 +1,113 @@
+import { PageText, TranslatedString, useTranslation } from '@atb/translations';
+import { SectionCard } from '../components/section-card';
+import { Input } from '../components/input';
+import style from '../contact.module.css';
+import { RadioInput } from '../components/input/radio';
+
+type FormSelectorProps = {
+  state: any;
+  send: any;
+};
+
+export const FormSelector = ({ state, send }: FormSelectorProps) => {
+  const { t } = useTranslation();
+
+  return (
+    <div>
+      {!state.context.isIntialAgreementChecked && (
+        <SectionCard title={PageText.Contact.travelGuarantee.title}>
+          <p>
+            {t(
+              PageText.Contact.travelGuarantee.layoutAgreement
+                .delayedRefundText,
+            )}
+          </p>
+
+          <p>
+            {t(
+              PageText.Contact.travelGuarantee.layoutAgreement.ticketRefundText,
+            )}
+          </p>
+
+          <h4>
+            {t(
+              PageText.Contact.travelGuarantee.layoutAgreement
+                .travelGuaranteeExceptions.label,
+            )}
+          </h4>
+          <p>
+            {t(
+              PageText.Contact.travelGuarantee.layoutAgreement
+                .travelGuaranteeExceptions.minimumTimeToNextDeparture,
+            )}
+          </p>
+
+          <p>
+            {t(
+              PageText.Contact.travelGuarantee.layoutAgreement
+                .travelGuaranteeExceptions.externalFactors,
+            )}
+          </p>
+          <ul className={style.rules__list}>
+            {PageText.Contact.travelGuarantee.layoutAgreement.travelGuaranteeExceptions.examples.map(
+              (example: TranslatedString, index: number) => (
+                <li key={index}>{t(example)}</li>
+              ),
+            )}
+          </ul>
+
+          <p>
+            {t(
+              PageText.Contact.travelGuarantee.layoutAgreement
+                .travelGuaranteeExceptions.exclusion,
+            )}
+          </p>
+
+          <Input
```

Comment:
Should use the new `Checkbox` component to have a similar appearance.

Reply:
👍